Tasks/ServiceFabricDeploy/task.json

```diff
@@ -338,34 +339,46 @@
             "type": "connectedService:AzureRM",
             "label": "Azure subscription",
             "helpMarkDown": "Select an Azure subscription",
-            "visibleRule": "configureDockerSettings = true && containerregistrytype = Azure Container Registry",
+            "visibleRule": "configureDockerSettings = true && registryCredentials = AzureResourceManagerEndpoint",
             "required": true,
             "groupname": "docker"
         },
         {
-            "name": "azureContainerRegistry",
```

Comment:
Don't we still need this? Or do we only need the credentials at this point?

Reply:
No, it's not needed.  We only need the credentials, just like the ServiceFabricComposeDeploy task.

---
In reply to: [136673349](https://github.com/Microsoft/vsts-tasks/pull/5076#discussion_r136673349) [](ancestors = 136673349)